### PR TITLE
Add project for testing CIFuzz changes

### PR DESCRIPTION
**Pull request by jonathanmetzman**

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
TBR

File: projects/cifuzz-example/project.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+language: c++
+sanitizers:
+  - address
+
```
Comment (Dor1s) [inline projects/cifuzz-example/project.yaml:4]: did you also want to disable other fuzzing engines?

Comment (jonathanmetzman) [reply projects/cifuzz-example/project.yaml:4]: Will it get built by the builders? I thought it won't because it is disabled.

Comment (Dor1s) [reply projects/cifuzz-example/project.yaml:4]: The builders shouldn't build it at all. I was thinking about CIFuzz, but sounds like we don't use fuzzing engines other than libFuzzer, right?

Comment (jonathanmetzman) [reply projects/cifuzz-example/project.yaml:4]: that's correct.

**Pull request merged**